wavefront_cli/lib/agent.py

```diff
@@ -1,63 +1,76 @@
-import subprocess
+"""Manage Telegraf agent."""
+
+
 import sys
 
-import message
-import system
-import aws
+from . import message
+from . import system
 
-agent_pkg_deb = "https://packagecloud.io/install/repositories/wavefront/telegraf/script.deb.sh"
-agent_pkg_rpm = "https://packagecloud.io/install/repositories/wavefront/telegraf/script.rpm.sh"
-telegraf_conf = "https://raw.githubusercontent.com/wavefrontHQ/integrations/master/telegraf/telegraf.conf"
-conf_path = "/etc/telegraf/telegraf.conf"
+
+CONF_PATH = "/etc/telegraf/telegraf.conf"
 
 
 def get_install_agent_cmd():
+    """Get OS specific command to install Telegraf agent."""
+    agent_pkg_deb = "https://packagecloud.io/install/repositories/" \
+                    "wavefront/telegraf/script.deb.sh"
+    agent_pkg_rpm = "https://packagecloud.io/install/repositories/" \
+                    "wavefront/telegraf/script.rpm.sh"
     dist = system.check_os()
+    cmd = None
     if not dist:
-        print "Error: Unsupported OS version. Please contact support@wavefront.com."
-        return None
+        print("Error: Unsupported OS version. Please contact"
+              " support@wavefront.com.")
+        return cmd
+
+    # pylint: disable=R0916
     if dist == "Oracle Linux Server" or dist.strip() == "Fedora" or \
-            dist == "Red Hat Enterprise Linux Server" or dist == "Red Hat Enterprise Linux Workstation" or \
-            dist == "CentOS" or dist == "CentOS Linux" or dist.startswith("Amazon Linux"):
+            dist == "Red Hat Enterprise Linux Server" or\
+            dist == "Red Hat Enterprise Linux Workstation" or \
+            dist == "CentOS" or dist == "CentOS Linux" or\
+            dist.startswith("Amazon Linux"):
```

Comment:
It might be simplified
```suggestion
            dist in ("Oracle Linux Server", "Red Hat Enterprise Linux Server" ..., "CentOS Linux) or dist.startswith("Amazon Linux"):
```